client/lib/route/page-notifier.js

```diff
@@ -1,23 +1,34 @@
 /*eslint no-multi-spaces: 0*/
 // node dependency
 var EventEmitter   = require( 'events' ).EventEmitter;
+var ipc = require( 'electron' ).ipcRenderer;
+var includes = require( 'lodash' ).includes;
 
 // internal dependency
-var page           = require( 'page' );
-var Route          = page.Route;
+var page		   = require( 'page' );
+var Route		  = page.Route;
 
 // routing state and middleware
-var location       = new EventEmitter();
+var location	   = new EventEmitter();
 var currentContext = null;
-var route          = new Route( '*' );
+var commandLine	= {};
+var route		  = new Route( '*' );
 
 // Inject our route at the front so we are always called
 page.callbacks.unshift( route.middleware( function( context, next ) {
+	context.commandLine = commandLine;
```

Comment:
Is this sensitive to the order in which things are called? By that I mean the command line will only be sent after the page has loaded (`did-finish-load`). At this point could this code already have run and the command line be empty? I haven't tested it out to see whether this runs before or after the page has loaded.